ansible/roles/debops.netbox/defaults/main.yml

```diff
@@ -40,15 +40,15 @@ netbox__domain: '{{ ansible_local.core.domain
 netbox__base_packages:
   - 'git'
   - 'build-essential'
-  - 'python-dev'
-  - 'python-pip'
+  - 'python{{ netbox__virtualenv_version }}-dev'
+  - 'python{{ netbox__virtualenv_version }}-pip'
```

Comment:
The `netbox.yml` playbook already contains the `debops.python` role which is required by the `debops.gunicorn` role. You could move the `python*` APT packages to the `netbox__python__dependent_packages2` and `netbox__python__dependent_packages3`, so that the `debops.python` role itself installs the desired version (see how it's done in `debops.gunicorn` role for an example). You would probably only need to install `python-dev`/`python3-dev` packages since `pip` and `virtualenv` is already included in `debops.python` role by default.

Reply:
This should now be implemented.